tests/core.py

```diff
@@ -1163,6 +1163,18 @@ def test_fetch_task_instance(self):
         response = self.app.get(url)
         assert "runme_0" in response.data.decode('utf-8')
 
+    def test_task_info(self):
+        url_template = '/admin/airflow/task_info?dag_id={}&task_id={}'
```

Comment:
I think this leads to too much duplicated code and too much clutter for a simple test